mad-dog/src/helpers.js

```diff
@@ -55,6 +56,57 @@ const addAndUpgradeUsers = async (
   const existingUserIds = []
   const walletIndexToUserIdMap = {}
 
+  await _addUsers({ userCount, executeAll, executeOne, existingUserIds, addedUserIds, walletIndexToUserIdMap })
+  await _upgradeToCreator(executeAll, executeOne, numCreatorNodes)
```

Comment:
i like what you did here, lets be explicit that this upgrades all users to creator not just one

Reply:
yup! in `_upgradeToCreator()` executes the upgrade for all libs instances passed in, which correlates to the num of users

Reply:
sorry i meant that lets make the function name more explicit